cdap-data-fabric/src/test/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueTest.java

```diff
@@ -332,7 +334,10 @@ public void testPrefix() {
 
   @Override
   protected void forceEviction(QueueName queueName) throws Exception {
-    byte[] tableName = Bytes.toBytes(((HBaseQueueClientFactory) queueClientFactory).getTableName(queueName));
+    // This functionality (internal-to-cdap tableName -> hbase tableName conversion) should be localized
+    // It's also used in HBaseQueueTest#getConsumerConfigCache
+    String tableNameString = ((HBaseQueueClientFactory) queueClientFactory).getTableName(queueName);
```

Comment:
Should change `getTableName` to `getTableId`. We should get rid of the `TableId.from(String)` method.